sys/include/net/sixlowpan/nd.h

```diff
@@ -115,7 +115,7 @@ extern "C" {
 /**
  * @brief   Number of address registration retries
  *
- * @note    May not be greater than 7.
+ * @note    Must not be greater than 7.
```

Comment:
dito here, I haven't found any reference on a maximum [RFC 6775, 3.3](https://tools.ietf.org/html/rfc6775#section-3.3):

> The response to an address registration might not be immediate, since
   in route-over configurations the 6LR might perform Duplicate Address
   Detection against the 6LBR.  A host retransmits the Address
   Registration Option until it is acknowledged by the receipt of an
   Address Registration Option.

Reply:
(The same answer as for https://github.com/RIOT-OS/RIOT/pull/8277#discussion_r160623422 applies here)